lib/python/pcm_placeholder.py.in

```diff
@@ -15,4 +15,4 @@ that has to be configured by CMake
 
 """
 
-PCMSOLVER_PARSE_DIR='@PCMSOLVER_PARSE_DIR@'
+PCMSolver_PARSE_DIR='@PCMSolver_PARSE_DIR@'
```

Comment:
Note that this does break equality of psi4-top-dir/lib and installed/share/psi/lib equality for the first time.

This isn't necessarily bad, as it had to happen some time. (Reasonably, add-ons may want to install into driver or lib.) But it does mean that running from uninstalled objdir with PSIDATADIR isn't equiv to running from installed. Just be aware.